docs/user/build-customization.rst

```diff
@@ -105,24 +105,14 @@ To avoid this, it's possible to unshallow the clone done by Read the Docs:
          - git fetch --unshallow
 
 
-Skip build based on a condition
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Cancel build based on a condition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There may be situations where you want to skip a build that was automatically triggered when someone on your team pushed to the repository.
-Skipping builds will allow you to speed up review times and also help us reduce server costs and ultimately our environmental footprint.
-Consider the following scenarios:
+When a command exits with code ``439``,
+Read the Docs will cancel the build immediately.
+We can use this approach to cancel builds we don't want to complete based on a condition.
```

Comment:
Who is `We`? I think we want to talk to the user right?

```suggestion
You can use this approach to cancel builds that you don't want to complete based on some conditional logic.
```

Reply:
Yes, "we" is incorrect here.